src/main/java/com/butbetter/applicationservices/ApplicationServices.java

```diff
@@ -1,4 +1,4 @@
-package com.butbetter.applicationServices;
```

Comment:
why did you change the name of the package?

Reply:
codacy issue